extensions/nodes/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nodes",
+  "title": "Nodes",
```

Comment:
Is this the best name for the extension?

Reply:
A node is either a RPC or a Validator on Solana, so the name is works fine imo. If you have strong feelings against it, you feel it is misleading or you feel like it should be more descriptive (Solana Nodes, Solana Validator Nodes, ...), pls let me know. 🫡